src/test/java/io/github/externschool/planner/repository/RoleRepositoryIntegrationTest.java

```diff
@@ -0,0 +1,56 @@
+package io.github.externschool.planner.repository;
+
+import io.github.externschool.planner.entity.Role;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class RoleRepositoryIntegrationTest {
+    @Autowired
+    private RoleRepository roleRepository;
+
+    @Test
+    public void testTableContainsPresetRoles() {
+        List<String> names = new ArrayList<>(Arrays.asList("admin", "guest", "officer", "student", "teacher"));
+        List<Role> roles = readRoles();
+
+        assertThat(roles).isNotEmpty();
+        assertThat(roles.size() == 5);
```

Comment:
this code is too complicated to read
for example, the following code will be more understandable

```
assertThat(roles)
 .isNotEmpty()
 .hasSize(5)
 .containsExactlyInAnyOrder(
   new Role("admin"), 
   new Role("guest"), 
   new Role("officer"), 
   new Role("student")
)

```